CHANGELOG.md

```diff
@@ -10,7 +10,7 @@ Each release contains a link for "Full Changelog"
 ## 0.1.14
 
 ### Added
-
+- ([#254](https://github.com/microsoft/hi-ml/pull/254)) Automatically rescale the learning rate by the number of GPUs available.
```

Comment:
Just FYI, we don't need to update changelog anymore

Reply:
shall I remove this then? Or can it stay?